db/migrate/20180608190543_add_lastmod_to_searchgov_url.rb

```diff
@@ -0,0 +1,5 @@
+class AddLastmodToSearchgovUrl < ActiveRecord::Migration
+  def change
+    add_column :searchgov_urls, :lastmod, :datetime
```

Comment:
Achtung!

This column is being added to the `searchgov_urls` table, so this migration could take a while.

Reply:
Thanks for the heads up.

How would you feel about naming the new column `last_modified_at` so that it is more consistent with the other multi-word columns in this table?

* `last_crawled_at`
* `last_crawl_status`
* `load_time`
* `created_at`
* `updated_at`
* `searchgov_domain_id`

Reply:
@noremmie , I've thought about this more and I would like to keep this consistent with the `lastmod` sitemap attribute. I've tested this on demo1, and it's working as desired. Let me know if you'd like further testing on this; otherwise, I'll go ahead and merge it.

Reply:
(P.S. My first reply didn't get posted immediately. Despite the timestamps, several hours did pass between those two comments!)

Reply:
Oh - I see. That makes sense.